cli/lsp/testing/collectors.rs

```diff
@@ -495,6 +496,37 @@ impl Visit for TestCollector {
                   );
                 }
               }
+            } else if ns_prop_ident_name == "ignore"
+              || ns_prop_ident_name == "only"
+            {
+              if let ast::Expr::Member(child_member_expr) =
+                member_expr.obj.as_ref()
+              {
+                if let ast::MemberProp::Ident(ns_prop_ident) =
+                  &child_member_expr.prop
+                {
+                  let ns_prop_ident_name = ns_prop_ident.sym.to_string();
+                  if ns_prop_ident_name == "test" {
+                    if let ast::Expr::Ident(ident) =
+                      child_member_expr.obj.as_ref()
+                    {
+                      if ident.sym.to_string() == "Deno" {
+                        visit_call_expr(
+                          node,
+                          Some(&self.fns),
+                          source_range_to_lsp_range(
+                            &ns_prop_ident.range(),
+                            &self.text_info,
+                          ),
+                          None,
+                          &self.text_info,
+                          &mut self.test_module,
+                        );
```

Comment:
This `visit_call_expr` is the same as above. Perhaps these expressions could return an optional value that's instead used by a common call to `visit_call_expr`.